data-processing/tests/common/commands/test_fetch_s2_data.py

```diff
@@ -14,6 +14,9 @@
 )
 from scripts.commands import run_command
 
+references_fields = ["authors", "title", "externalIds", "venue", "year"]
```

Comment:
What do you think of maybe calling these `expected_references_fields` and `expected_fields_query`?